### ISSUE TITLE
is it possible to style or add border to empty cell?

### ISSUE THREAD

**Issue by laptopmutia** [axlsx-styler-gem/axlsx_styler#33]
or the only way is to hack the cell with `''` ?

I build my spreadsheet with empty row and nil row ( is that good practices?)

```
  sheet.add_row ['Consignee', nil, ": #{@job.company.name}", nil, nil, nil, "Vessel", ": #{@job.vessel_name}"], offset: 1
  sheet.add_row
  sheet.add_row ["DESCRIPTION", nil, nil, nil, "QTY", "UNIT", "CHARGE", "NET VALUE"], offset: 1
  14.times do
    sheet.add_row
  end
  sheet.add_row ["Payment Term : COD", nil, nil, nil, nil, nil, "SUB TOTAL", "JPY"], offset: 1
```

whenever I try to add style or border I got this
```undefined method `row' for nil:NilClass```

relevant issues:
https://github.com/axlsx-styler-gem/axlsx_styler/issues/26
https://github.com/axlsx-styler-gem/axlsx_styler/issues/27
https://github.com/randym/axlsx/issues/586

**Comment by westonganger** [comment]
https://github.com/axlsx-styler-gem/axlsx_styler/issues/27 seems to suggest that you are trying to add a style on a row number that is too high. Is this the case here?

**Comment by westonganger** [comment]
Also a more complete stack trace would help too

**Comment by westonganger** [comment]
Closing this for lack of further response.

To improve the error message for the whole community, I created a [PR](https://github.com/caxlsx/caxlsx/pull/122) to caxlsx which was merged and released in caxlsx@3.2.0.
